### ISSUE TITLE
gtest archive link 404

### ISSUE THREAD

**Issue by pcj** [livegrep/livegrep#30]
The zip for gtest-1.7.0 seems broken to me.  I updated rules_protobuf to use a newer build file that depends on github instead of googlecode.  If you'd like to stay with 1.7.0, the updated link is https://github.com/google/googletest/archive/release-1.7.0.zip.  If you'd like to move to gtest 1.8, here's a link to the BUILD file I'm using: https://github.com/pubref/rules_protobuf/blob/master/bzl/build_file/gtest.BUILD

**Comment by nelhage** [comment]
Hm, the link still seems to work for me:

```
[nelhage@monomorphique:/tmp]$ curl -sLO https://googletest.googlecode.com/files/gtest-1.7.0.zip
[nelhage@monomorphique:/tmp]$ sha256sum gtest-1.7.0.zip
247ca18dd83f53deb1328be17e4b1be31514cedfc1e3424f672bf11fd7e0d60d  gtest-1.7.0.zip
[nelhage@monomorphique:/tmp]$ file gtest-1.7.0.zip
gtest-1.7.0.zip: Zip archive data, at least v1.0 to extract
```

are you seeing a failure?

But I'll bump to use the github URL anyways since that's probably going to be around longer.

**Comment by nelhage** [comment]
2e7fff634462ce8c6561a95daf60731d0749dc0a will cut over to just using your external, since I'm planning on  pulling in `rules_protobuf`. I don't think I'll bother fixing on master until that lands.

**Comment by gnprice** [comment]
It looks like 2e7fff6 has landed in master, and then 60df22c9d switched the remote. Is this resolved now?

**Comment by nelhage** [comment]
Should be.
